mappings/net/minecraft/client/render/block/entity/BannerBlockEntityRenderer.mapping

```diff
@@ -11,3 +11,4 @@ CLASS net/minecraft/class_823 net/minecraft/client/render/block/entity/BannerBlo
 		ARG 5 baseSprite
 		ARG 6 isBanner
 		ARG 7 patterns
+	METHOD method_32135 getTexturedPartData ()Lnet/minecraft/class_5607;
```

Comment:
Would these update too along with the class rename?

Reply:
yes 😢